.gitignore

```diff
@@ -3,3 +3,4 @@ cloudmailer/cloudmailer.cfg
 *.bak
 cloudmailer/temporary_files
 node-drainer/log_files/
+cloudmailer/templates/my_*
```

Comment:
Why you ignore templates that starts only with my_

Reply:
so there can be other templates in the public repository but keep the ones starting with my_* private

Reply:
You may need to update the readme of cloudmailer.